src/store/createStore.js

```diff
@@ -0,0 +1,118 @@
+'use strict'
+
+import assign from 'object-assign'
+import AltStore from '../../src/store/AltStore'
```

Comment:
I think we need to update some of the build scripts so that we can do `import AltStore from './store/AltStore'` etc.

Wasn't sure of the best way to do this with browserify (especially since the build files need to be consumable by the user)